src/components/data_search/DatasetSearchTable.js

```diff
@@ -40,40 +43,121 @@ export const DatasetSearchTable = (props) => {
   const [selected, setSelected] = useState([]);
   const [exportableDatasets, setExportableDatasets] = useState({}); // datasetId -> snapshot
   const [tdrApiUrl, setTdrApiUrl] = useState('');
-
+  const searchRef = useRef('');
   const isFiltered = (filter) => filters.indexOf(filter) > -1;
 
-  const filterHandler = (event, data, filter) => {
+  const assembleFullQuery = (searchTerm, filters) => {
+    const queryChunks = [
+      {
+        'match': {
+          '_type': 'dataset'
+        }
+      },
+      {
+        'exists': {
+          'field': 'study'
+        }
+      }
+    ];
+  
+    // do not apply search modifier if there is no searchTerm
+    if (searchTerm !== '') {
+      const searchModifier = [
+        {
+          "multi_match": {
```

Comment:
Right now this matches only as a prefix, but this is a good start, so perhaps we can add a ticket to the backlog to investigate how to search for whole phrases.